f21/ps.html

```diff
@@ -1066,57 +1075,54 @@ <h2><u> Lecture / Lab 8</u></h2>
 
 
 
-  <li > <b>Due Date: 12 November</b>
+  <li > <b>Due Date: 5 November</b>
   <b style="float:right">Reading: <a href="https://pandas.pydata.org/pandas-docs/stable/10min.html">10-mins to Pandas</a>, <a href="https://www.datacamp.com/community/tutorials/pandas-tutorial-dataframe-python#gs.hnyRaik">DataCamp Pandas</a> & <a href="lab7.html">Lab 7</a> & <a href="lab8.html">Lab 8</a> </b><br>
 
 
 
-<p> Write a program that asks the user for a CSV of Dinosaur Genus data:
+<p> Write a program that asks the user for a CSV of Video Games Dataset
 <ul>
    <li> There is a sample file on <a href="https://github.com/HunterCSci127/CSci127">github</a>.
 </ul>
 
 Your program should then print out:
   <ul>
-    <li> The total number of genera (in this dataset each row is a dinosaur genus, genera is the plural of genus)
-    <li> The count of dinosaur genera for each period
-    <li> The three most populated countries.
+    <li> The total number of games
+    <li> The count of video games of each game genre. 
+    <li> The top 3 game publishers
   </ul>
 
 <p>A sample run:
 <pre style = "background-color:lavender;width:500px">
-Enter file name: dinosaur.csv
-There are 1154 dinosaur genera.
-
-The number of dinosaur genera in each period:
-Cretaceous               763
-Jurassic                 265
-Triassic                  50
-Late Cretaceous           30
-Early Cretaceous          16
-Early Jurassic             7
-Jurassic/Cretaceous        7
-Middle Jurassic            4
-Late Triassic              3
-Triassic or Jurassic       3
-Triassic/Jurassic          3
-Cretaceous                 1
-(unknown)                  1
-Early-Late Cretaceous      1
-Name: Period, dtype: int64
-
-
-The three most dinosaur-populated countries were:
-North America    274
-China            217
-South America    149
-Name: Country, dtype: int64
+Enter file name: vgsales.csv
+There are 16598 total games
+
+The number of game genres are 
```

Comment:
Please make this "The number of games in each genre are:"
Please update the autograder  accordingly too if necessary. Thank you!